### DISCUSSION TITLE
Connect a Wireguard Instance

### DISCUSSION THREAD

**Discussion by DoctorKisow** [eduardogsilva/wireguard_webadmin#39] [Help and Support]
I am running Wireguard as a standalone instance on a 22.04 LTS server, how do you connect the web-admin to that instance, what am I missing?

**Comment by eduardogsilva** [comment]
Hello @DoctorKisow , you can't connect the webadmin to an existing wireguard instance. 
You need to install wireguard_webadmin, it will provide it's own instance.
Just follow the [install instructions](https://github.com/eduardogsilva/wireguard_webadmin?tab=readme-ov-file#deployment-instructions)
Cheers!

**Reply by DoctorKisow** [reply]
I figured that out by interrogating the container; firewall rules had me guessing a bit, but I figured them out too.  Thanks! Great project, nice work!

**Reply by eduardogsilva** [reply]
Glad to hear! Let me know if you need anything else.

Cheers!
